dubbo-native/src/main/java/org/apache/dubbo/aot/generate/AotProcessor.java

```diff
@@ -51,6 +51,7 @@ public static void main(String[] args) {
             .registerAdaptiveType(new ArrayList<>(ClassSourceScanner.INSTANCE.adaptiveClasses().values()))
             .registerBeanType(ClassSourceScanner.INSTANCE.scopeModelInitializer())
             .registerConfigType(ClassSourceScanner.INSTANCE.configClasses())
+            .registerBeanType(ClassSourceScanner.INSTANCE.getNetty4ChannelHandlerSubclasses()) // actually queryAllPublicMethods is enough
```

Comment:
It's not supposed to handle Netty content here.

Reply:
I need the jar scanner,if move to the ReflectionTypeDescriberRegistrar can scanner expose to it？